src/ucp/core/ucp_ep.h

```diff
@@ -350,11 +350,20 @@ typedef struct {
 } ucp_ep_ext_proto_t;
 
 
+enum {
+    UCP_WIREUP_SOCKADDR_CD_FULL_ADDR = 0, /* Client data contains full address */
+    UCP_WIREUP_SOCKADDR_CD_PARTIAL_ADDR,  /* Client data contains partial address,
+                                             wireup protocol requires extra MSGs */
+    UCP_WIREUP_SOCKADDR_CD_LOCAL_ADDR     /* Client data contains locally built address */
```

Comment:
what does it mean locally? built by CM?
maybe "UCP_WIREUP_SOCKADDR_CD_CM_ADDR"      ?